### PR TITLE
adding gap between form elements skill and knowledge

### PR DESCRIPTION
**Pull request by ashapadmashetti**
closes : https://github.com/instructlab/ui/issues/149

### CODE REVIEW

File: src/components/Contribute/Knowledge/AttributionInformation/AttributionInformation.tsx [hunk 1]
```diff
@@ -169,11 +169,13 @@ const AttributionInformation: React.FC<Props> = ({
           <FormHelperText>
             <HelperText>
               <HelperTextItem icon={<ExclamationCircleIcon />} variant={validLink}>
-                Please enter a valid URL.
+                Required field
```
Comment (vishnoianil) [inline src/components/Contribute/Knowledge/AttributionInformation/AttributionInformation.tsx:172]: I think we need to update this text, given that it doesn't only check if the box is empty, but it also checks if the entered string is valid string. I think, let's rephrase it to "URL is required and must be a valid URL".

**Review by vishnoianil — CHANGES_REQUESTED** [review]
Looks good to me. One minor comment. Also can you please cherry-pick it to the release-1.0 branch as well

File: src/components/Contribute/Knowledge/AttributionInformation/AttributionInformation.tsx (hunk 1 above)
Comment (ashapadmashetti) [reply src/components/Contribute/Knowledge/AttributionInformation/AttributionInformation.tsx:172]: Sure I will update that
